src/main/java/edu/tamu/scholars/middleware/export/service/CsvExporter.java

```diff
@@ -121,11 +121,20 @@ private List<Object> getRow(
             if (content.containsKey(property)) {
 
                 Object value = content.get(property);
+                if (property.equals("type")) {
+
+                    List<String> values = asStringList(value);
+
+                    if (!values.isEmpty()) {
+                        value = values.stream()
+                            .map(this::formatPersonType)
+                            .toList();
+                    }
+                }
 
                 if (List.class.isAssignableFrom(value.getClass())) {
 
-                    @SuppressWarnings("unchecked")
-                    List<String> values = (List<String>) value;
+                    List<String> values = asStringList(value);
```

Comment:
```suggestion
                    List<String> values = asStringList(value);
```
I suggest this style clean up by removing the extra line.

Reply:
Unable to apply this suggestion.